Source/Csla.Web.Mvc.Shared/Server/Hosts/HttpPortalController.cs

```diff
@@ -173,6 +173,8 @@ private async Task InvokePortal(string operation, Stream requestStream, Stream r
       var serializer = new MobileFormatter();
       var result = new HttpResponse();
       HttpErrorInfo errorData = null;
+      Response.Headers.Add("Content-type", "application/octet-stream,text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9");
```

Comment:
Does this apply if the use-text-serialization flag is set for Blazor? In that case the byte stream MUST BE just text, because of limitations of webassembly.

Reply:
As far I have seen Blazor on requests adds all of these types.

And if our header response is not matched, that data will not be displayed.

I remember when I only had headers with Content type application-octet-stream, with no compression, Blazor did not work at all.

I will double check webasembly in the morning, to match all the headers.

Reply:
Thank you. It is Blazor WebAssembly that I'm concerned about.

Reply:
Hi Rocky, I have tested WebAssembly and it should work, it accepts all headers, and don't see any reason why it should not work.
![client](https://user-images.githubusercontent.com/34435929/98336810-dd9f5a00-2007-11eb-8d13-0e5af249a477.jpg)


However, making prerelease 5.4 would make my testing whole lot simpler.  I don't think we will break anyone's installation, and we can make the proper ProjectTracker/CSLA adjustments for production version.